src/main/java/de/muenchen/isi/domain/service/calculation/CalculationService.java

```diff
@@ -318,4 +349,30 @@ public LangfristigerSobonBedarfModel calculateLangfristigerSobonursaechlicherBed
 
         return bedarf;
     }
+
+    public boolean hasRightCoonditionsForLangfristigeSobonBerechnung(
```

Comment:
Der Methodenname sagt nichts über die fachliche Logik aus.

Der Methodenname sollte dem entsprechen was in der Methode geprüft wird.

evtl `doSobonberechung`?